app/src/main/java/org/mozilla/fenix/library/history/HistorySearchDialogFragment.kt

```diff
@@ -170,6 +179,7 @@ class HistorySearchDialogFragment : AppCompatDialogFragment(), UserInteractionHa
         consumeFrom(store) {
             toolbarView.update(it)
             awesomeBarView.update(it)
+            addVoiceSearchButton()
```

Comment:
I don't know if this really needs to be in here. The difference I see is that we aren't utilizing the search state. https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt#636

So, my question is do we care about anything from the search state currently?

Reply:
This doesn't consuming anything from the store anymore, should be moved out.  I'll do that thanks